docs/getting-started/README.md

```diff
@@ -1,7 +1,13 @@
 # Getting started with the Forms System Library
 
-### Creating a new application with the US Forms System Starter App
+Depending on your needs, you can either create a brand new application from scratch or install the US Forms Library in an existing application.
 
-### Installing the library in an existing application
+### [Creating a new application with the US Forms System Starter App](getting-started/creating-a-new-application-with-the-us-forms-system-starter-app.md)
+
+The US Forms System Starter App creates the initial files, configuration, build process, and web server you need to start building your form immediately.
+
+### [Installing the library in an existing application](getting-started/installing-the-library-in-an-existing-application.md)
+
+If your existing application has a web server to run the app locally, you can install the US Forms System Library in that app.
```

Comment:
I think the real distinction of why you would want to go this route is less about the existence of a web server and more about just having an existing React app that you want to integrate us-forms-system into, so I might change this content to reflect that. Let me know if that doesn't make sense!

Reply:
That makes a ton of sense, and is a much better description. I'll incorporate that!